phone_iso3166/e212.py

```diff
@@ -144,38 +425,35 @@
     422: "OM",
     424: "AE",
     425: {
-        1: ("IL"),
-        2: ("IL"),
-        3: ("IL"),
-        4: ("IL"),
-        5: ("PS"),
-        6: ("PS"),
-        7: ("IL"),
-        8: ("IL"),
-        9: ("IL"),
-        11: ("IL"),
-        12: ("IL"),
-        13: ("IL"),
-        14: ("IL"),
-        15: ("IL"),
-        16: ("IL"),
-        17: ("IL"),
-        18: ("IL"),
-        19: ("IL"),
-        20: ("IL"),
-        21: ("IL"),
-        23: ("IL"),
-        24: ("IL"),
-        26: ("IL"),
-        28: ("IL"),
-        29: ("IL"),
+        1: "IL",
+        2: "IL",
+        3: "IL",
+        4: "IL",
+        5: "PS",
+        6: "PS",
+        7: "IL",
+        8: "IL",
+        9: "IL",
+        10: "IL",
+        12: "IL",
+        16: "IL",
+        18: "IL",
+        19: "IL",
+        20: "IL",
+        21: "IL",
+        22: "IL",
+        23: "IL",
+        24: "IL",
+        25: "IL",
+        26: "IL",
+        27: "IL",
+        28: "IL",
+        29: "IL",
     },
     426: "BH",
     427: "QA",
     428: "MN",
     429: "NP",
-    430: "AE",
-    431: "AE",
```

Comment:
These do exist, but as far as I can tell there is no networks to this MCC yet, so the wiki pages does not have any networks, and therefore they are skipped.

Should we have a list of additional MCC to countries we'd like to bolt on to this dataset @jwldk ?